src/components/cards/TradeCardGP/TradeCardGP.vue

```diff
@@ -20,6 +20,13 @@
         @amountChange="trading.handleAmountChange"
         class="mb-4"
       />
+      <GasReimbursement
+        v-if="!ENABLE_LEGACY_TRADE_INTERFACE && trading.isBalancerTrade.value"
+        class="mb-5"
+        :address-in="tokenInAddress"
+        :address-out="tokenOutAddress"
+        :sorReturn="trading.sor.sorReturn.value"
+      />
```

Comment:
I thought this was going to go when we made Gnosis the default? Even if we revert back to the Balancer interface I think the gas reimbursement program is coming to an end. @markusbkoch?

Reply:
Internally we discussed ending BAL for Gas when Gnosis was made the default. But the governance proposal predates those discussions, so we'd need a vote in order to end the program. We can do it this weekend.